ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListSourceExecutor.java

```diff
@@ -241,27 +262,52 @@ private ListSourceExecutor() {
         .collect(Collectors.toList());
   }
 
+  private static SourceDescriptionWithWarnings describeSource(
+      final KsqlConfig config,
+      final KsqlExecutionContext executionContext,
+      final ServiceContext serviceContext,
+      final SourceName table,
+      final boolean extended,
+      final ConfiguredStatement<ShowColumns> statement,
```

Comment:
Is this just visible for testing? It seems specific for it to take `ShowColumns`

Reply:
good point, it was me trying to get to grips with intellij's refactoring tools. but actually it makes more sense to change this signature rather than overload a private. Fixed!